asset/css/markbind.css

```diff
@@ -5,3 +5,12 @@ blockquote {
 .radio-list-item label {
     font-weight: inherit;
 }
+
+@media (max-width: 767px) {
+  .navbar-nav .open .dropdown-menu {
+    background-color: #fff;
+  }
+  .navbar-inverse .navbar-nav .open .dropdown-menu > li > a {
+    color: #000;
+  }
```

Comment:
[CSS standard: use 4 spaces for indentation](https://github.com/oss-generic/process/issues/50)

Reply:
Apologies, I made a force push to the branch with updated indentation after noticing it was off.